services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/AutoReroutesSpec.groovy

```diff
@@ -25,7 +25,10 @@ class AutoReroutesSpec extends BaseSpecification {
     @Autowired
     NorthboundService northboundService
 
-    def flowStatusTimeout = 7
+    @Value('${reroute.delay}')
```

Comment:
reroute.delay or reroute.timeout? According to the variable name 'rerouteTimeout' - it should be second variant.